Actually, the linter is complaining:

```

scripts/west_commands/sign.py:223:11: C0121: Comparison 'build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT') == False' should be 'build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT') is False' if checking for the singleton value False, or 'not build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT')' if testing for falsiness (singleton-comparison)
```

Please apply the change I suggested above.

--- a/scripts/west_commands/sign.py
+++ b/scripts/west_commands/sign.py
@@ -220,7 +220,7 @@ class ImgtoolSigner(Signer):
         flash = self.edt_flash_node(b, args.quiet)
         align, addr, size = self.edt_flash_params(flash)
 
-        if build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT'):
+        if not build_conf.getboolean('CONFIG_BOOTLOADER_MCUBOOT'):
             log.wrn("CONFIG_BOOTLOADER_MCUBOOT is not set to y in "
                     f"{build_conf.path}; this probably won't work")
 

